data/members/Kehinde Adeleke.json

```
@@ -0,0 +1,13 @@
+{
+  "date": "2023-07-14T00:00",
+  "name": "Kehinde Adeleke",
+  "url": "https://www.kehinde.xyz/",
+  "employment": {
+    "hiring": false,
+    "seeking": true
+  },
+  "linkedin": "https://linkedin.com/in/adeleke5140",
+  "mastodon": "https://github.com/adeleke5140",
```

```suggestion
  "mastodon": null,
```

Looks like we're using a GitHub URL for a non-GitHub related key here. I raised https://github.com/ericwbailey/a11y-webring.club/issues/120 and self-assigned it, so we could totally do a followup PR to add a GitHub profile URL once there is a proper place for it.

---

Thank you for noting the error, Eric. I checked the `json` file multiple times but still missed it.

---

No problem! These things happen 🙂